src/UI/Forms/Controls/AutocompleteFormControl.js

```diff
@@ -0,0 +1,18 @@
+import { Autocomplete, FormControl, FormHelperText } from "@mui/material";
+
+const AutocompleteFormControl = ({ hasError, error, ...otherProps }) => {
+  const formControlConfig = {
+    fullWidth: true,
+    error: hasError,
+  };
+  const helperTextStyle = hasError ? {} : { display: "none" };
+
+  return (
+    <FormControl {...formControlConfig}>
+      <Autocomplete {...otherProps} />
+      <FormHelperText style={helperTextStyle}>{error}</FormHelperText>
```

Comment:
FormControll is needed only for custom inputs

Actually, I'm okay with any rendering

But an error/hint message for Autocplete can be assigned via `renderInput`

https://stackoverflow.com/a/63281653/531322